scripts/main.js

```diff
@@ -1,2 +1,8 @@
-console.log('I RAN')
-# consol log comment
+let cxValue = 50
+const circle = document.getElementById('some-circle')
+circle.setAttribute('cx', cxValue);
+
+circle.addEventListener('click', () => {
```

Comment:
Here we make it do things when it's clicked.

How about we go ahead and move this into a function!?  
We need an `updateLocation` function that takes the new `cxValue` and sets the attributes.

Eventually we'll also pass in a `cyValue` too!

Reply:
If you want a little help with functions check out this pretty decent tutorial on js functions
https://www.youtube.com/watch?v=AY6X5jZZ_JE

It also goes into how we added the js file to the html file which is nice.